### ISSUE TITLE
Extra spaces generated in case the value of config is a string with separator ","

### ISSUE THREAD

**Issue by thanhbnq** [test-fullautomation/python-jsonpreprocessor#66]
Dear experts,

I would like to share my observation during my usecase with your great tool.
In my case, I have the configuration in .json file like:

```
{
    ${params}['dSUT']['patch'] : "patch0.xml,patch1.xml,patch2.xml,patch3.xml"
}
```

What we would expect from the output of variable **${params.dSUT.patch}** should be:
```
"patch0.xml,patch1.xml,patch2.xml,patch3.xml"
```
But we actually got:
```
"patch0.xml, patch1.xml, patch2.xml, patch3.xml"
```

The extra spaces seem to be automatically added into the string.
I would like to know that it is an **unexpected behavior (from my point of view)** 
or **expected behavior (means we have to take care of removing those extra spaces by our code)**

Looking forward your check and response.
Thank you very much in advance.

Thanh

**Comment by test-fullautomation** [comment]
Hi Son,
from my point of view this is a bug.  The content of a string mustn't be modified in any case.
Please check.

In general:
If the value is a string it is only allowed to work on it if a `${` sequence is inside the string. Otherwise the content mustn't be touched. Please implement accordingly. 
This is also valid for keys and arrays containing strings, ....

@HolQue: Please add a SelfTest for this use case(s).

Thank you,
 Thomas

**Comment by namsonx** [comment]
Hello Thomas, Hello Holger,

I had reproduced this issue and found the root cause of it.
I will provide the pull-request to fix this issue tomorrow.

Thank you,
Son

**Comment by namsonx** [comment]
Hello Thomas,

I created the pull-request #68  to fix this issue.

Thank you,
Son

**Comment by test-fullautomation** [comment]
Hi Son,
thank you for the pull-request. Please respond to my question.
Thank you,
 Thomas

**Comment by test-fullautomation** [comment]
solved and already merged with #68
